mass_editing/migrations/10.0.2.0.0/pre-migration.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (C) 2019 - Today: GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+def migrate(cr, version):
+    if not version:
+        return
+
+    # First remove the obsolete constraint created by the existence of the
```

Comment:
I think that if you change the model in the 10.0 (that is OK), it will break the migration script of the 13.0 revision. 
Maybe we should add a test in the pre-migration script of the module 13.0 testing for exemple, if mass_editing table exists. If yes, skip the migration. Don't you think ?

Maybe you should add it also here, if for example, this change is also backported in 8.0 and 9.0.

kind regards.

Reply:
@legalsylvain thanks for your answer.
It's a good advice. I'm going to work on it.

Regards